ui/client/components/graph/Graph.js

```diff
@@ -186,13 +186,29 @@ class Graph extends React.Component {
         this.handleInjectBetweenNodes(cellView)
       })
       .on("link:connect", (c) => {
+        this.disconnectPreviousEdge(c.model.id);
         this.props.actions.nodesConnected(
           c.sourceView.model.attributes.nodeData,
           c.targetView.model.attributes.nodeData
         )
       })
   }
 
+  disconnectPreviousEdge = (previousEdge) => {
+    const nodeIds = this.extractNodeIds(previousEdge);
```

Comment:
Why not unpack the array and show what is going on?

Reply:
Done